pymc3/__init__.py

```diff
@@ -28,9 +36,18 @@
 from .tests import test
 
 from .data import *
+import theano
```

Comment:
This looks good to go, but I would wrap the whole thing in a mangled function, so that the top-level namespace does not get too busy.


```python
def __set_theano_flags():
    import theano
    import platform
    ...

__set_theano_flags()
```

Reply:
That's a good idea.